docs/docs/themes.md

```diff
@@ -0,0 +1,83 @@
+---
+id: themes
+title: Themes
+sidebar_label: Themes
+---
+
+## oh-my-posh themes
+
+### [Agnoster](https://github.com/JanDeDobbeleer/oh-my-posh3/blob/main/themes/agnoster.json)
```

Comment:
I like the links at the bottom rather than inline. Although for this use-case, it might be easier to do it this way.

Reply:
Well, I noticed that *after* I wrote the file and it was extremely late already. I think the links could be done at the bottom, will save some space since there are 2 exactly same links per theme.